db_logger.py

```diff
@@ -24,7 +26,11 @@ def log_chat(self, sender, message, channel):
             return
 
         try:
-            self.cursor.execute("INSERT INTO chat_log (sender, message, channel, date) VALUES (%s, %s, %s, %s)",
+            if self.senders_whitelist is not None and sender in self.senders_whitelist:
```

Comment:
`if (self.senders_whitelist is not None and sender in self.senders_whitelist) or (self.senders_whitelist is None):`